apps/homescreen/js/app.js

```diff
@@ -401,14 +424,19 @@
           if (child.order !== -1 && child.order < container.order) {
             continue;
           }
-          this.icons.insertBefore(container, child,
-                                  this.iconAdded.bind(this, container));
+          this.icons.insertBefore(container, child);
```

Comment:
Can we simplify the `insertBefore` to remove the callback argument or is it used elsewhere?